Cargo.toml

```
@@ -245,12 +245,13 @@ codegen-units = 1
 inherits = "dev"
 debug-assertions = false
 
+[profile.bench]
+lto = true
+
 # Enable debug information specifically for memory profiling.
 # https://docs.rs/dhat/0.2.1/dhat/#configuration
-#
-# 2021-01-08: This would be nicer as a named profile, e.g. [profile.memory]
-# https://github.com/rust-lang/cargo/issues/6988
-[profile.bench]
+[profile.memory]
```

Consider calling it `bench-memory`, as "memory" isn't very meaningul.
```suggestion
[profile.bench-memory]
```